src/main/java/com/moodybluez/enterprise/service/EntryServiceStub.java

```diff
@@ -1,46 +0,0 @@
-package com.moodybluez.enterprise.service;
-
-import com.moodybluez.enterprise.dto.Date;
-import com.moodybluez.enterprise.dto.Entry;
-import org.springframework.stereotype.Component;
-
-import java.util.HashMap;
-import java.util.Map;
-
-import static java.lang.Integer.parseInt;
-
-
-@Component
-public class EntryServiceStub implements IEntryService {
-
-    Map<String, Entry> moodEntryMap = new HashMap<>();
-
-    @Override
-    public void saveEntry(Entry entry) {
-        moodEntryMap.put(entry.date.getDate(), entry);
-    }
-
-    @Override
-    public Entry fetchByDate(String date) {
-        Entry entry = new Entry();
-        entry.setMoodID(3);
-        entry.setReasonDesc("I laid in bed all day.");
-
-        Date dateObj = new Date();
-        dateObj.setDate("2/22/2021");
-
-        moodEntryMap.put(date, entry);
-
-        if(moodEntryMap.containsKey(date)){
-            return moodEntryMap.get(date);
-        }
-        else {
-            return null;
-        }
-    }
-
-    @Override
-    public Map<String, Entry> fetchAll() {
-        return moodEntryMap;
-    }
-}
```

Comment:
Removed these files, because the logic here should really be in the DAO layer, which I have created and that includes necessary methods created in these service layer files.